doc/policy/packages.md

```diff
@@ -0,0 +1,57 @@
+# Package Mempool Accept
+
+## Definitions
+
+A **package** is an ordered list of transactions, representable by a Directed Acyclic Graph (a
+directed edge exists between a transaction that spends the output of another transaction).
```

Comment:
I've added "connected." I hesitate to write "fully" because I have sometimes seen "fully connected" used interchangeably with [complete](https://en.wikipedia.org/wiki/Complete_graph).